airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -555,6 +559,22 @@ def execute(self, context: Context) -> str | None:
             self.client_request_token,
             self.tags,
         )
+        if self.deferrable:
+            timeout = (
+                timedelta(seconds=self.max_polling_attempts * self.poll_interval + 60)
```

Comment:
+1 with the constant idea. this `60` seems to appear in multiple places

Reply:
Honestly, I'm thinking that with https://github.com/apache/airflow/pull/31757 closed (by @Lee-W :smile: ), that 60 seconds is no longer necessary. I initially added it because the Trigger would not log an error when timing out, so I added 60 seconds so the Trigger had some time to exit gracefully. 
I haven't tested the new timeout feature myself, but if it works as intended, then I think we can leave the 60 seconds off. What do you guys think?

Reply:
But https://github.com/apache/airflow/pull/31757 will be released in 2.6.3 and the provider min airflow version is 2.4.0 so does not make sense to keep it?